x-pack/heartbeat/monitors/browser/synthexec/execmultiplexer.go

```diff
@@ -28,20 +28,20 @@ func (e ExecMultiplexer) writeSynthEvent(se *SynthEvent) {
 		e.currentJourney.Store(true)
 		e.eventCounter.Store(-1)
 	}
-	hasCurrentJourney := e.currentJourney.Load()
+	se.index = e.eventCounter.Inc()
+
 	if se.Type == "journey/end" || se.Type == "cmd/status" {
 		e.currentJourney.Store(false)
 	}
-
-	se.index = e.eventCounter.Inc()
-	if hasCurrentJourney {
-		e.synthEvents <- se
-	}
+	e.synthEvents <- se
```

Comment:
@andrewvc WDYT about this? We pipe all the events from Synthetics even where are no active journeys and try to create appropriate stderr or stdlog synthevents? 

Do you see any problem with this approach?